src/RosAria.cpp

```diff
@@ -492,6 +520,20 @@ void RosAriaNode::publish()
   position.header.frame_id = frame_id_odom;
   position.child_frame_id = frame_id_base_link;
   position.header.stamp = ros::Time::now();
+
+  if(position.twist.twist.linear.x  == 0 && 
+     position.twist.twist.linear.y  == 0 && 
+     position.twist.twist.angular.z == 0)
```

Comment:
Wanted to be sure that the robot is 100\% stationary before sending good covariance values.